lambdas/service/app.py

```diff
@@ -1392,6 +1392,16 @@ def generate_manifest(event, context):
     return {'Location': presigned_url}
 
 
+file_version_parameter = params.query(
+    'version',
+    schema.optional(str),
+    description='''
+        The version of the file to be returned. File versions are opaque strings with only one documented property:
```

Comment:
Wrap on 80 and use `format_description`, please.

Reply:
Q: Should we be using `format_description` with triple strings in documentation? There appears to be mixed usage in the final spec for `/dss/files/{uuid}`; the parameter spec `dss_files_spec` not using the `format_description`, while response spec using `format_description`.